src/python_be.h

```diff
@@ -247,6 +250,10 @@ class ModelState : public BackendModel {
   // Validate Model Configuration
   TRITONSERVER_Error* ValidateModelConfig();
 
+  // Overrides `BackendModel::SetModelConfig` to also
```

Comment:
I believe in this case, it is more like: // Hides `BackendModel::SetModelConfig` ... ,
since base class' function is not virtual. Happy to change

Reply:
No.. This looks fine to me.